scripts/pack/README.md

```diff
@@ -0,0 +1,46 @@
+# Terra Toolkit Pack
+
+Terra Toolkit offers a way to pack individual packages into a tar archive file.
```

Comment:
In your prototype PR in terra-clinical, you updated the compile script to run 
```
"compile": "npm run compile:clean && npm run compile:build && tt-pack",
```
I know this is kinda a given but I wonder if its worth mentioning if this packs the whole package with src and lib, or just lib, if we need to compile first with babel etc.

Reply:
I added a note that assets should be compiled before running this command. Pack works just like releasing your package. It will pack everything in the package except the items blacklisted in the npmignore. 

https://github.com/cerner/terra-toolkit/pull/198/commits/2481a6e49cead4eba3e9bbdec6344d1009f41be6

Reply:
> It will pack everything in the package except the items blacklisted in the npmignore.

Lets add that as well. This would be handy to know if people were curious what a new package would be released with.

Reply:
https://github.com/cerner/terra-toolkit/pull/198/commits/78bcb62cf71077a3e462da7e6cfbda93d45c274b